editor/src/templates/editor-canvas.tsx

```diff
@@ -1592,11 +1593,21 @@ export class EditorCanvas extends React.Component<EditorCanvasProps> {
         // on macOS it seems like alt prevents the 'paste' event from being ever fired, so this is dead code here
         // needs testing if it's any help for other platforms
       } else {
+        const canvasWrapperRect = this.canvasWrapperRef?.getBoundingClientRect() ?? zeroCanvasRect
+        const offset = canvasPoint({
```

Comment:
for readability would it be better to call this `canvasViewportCenter[offset]`?

Reply:
great idea! Renamed it to `canvasViewportCenter` in https://github.com/concrete-utopia/utopia/pull/3693/commits/a4e26a0d131a8f33f34cd74a7c170f77e090c244